mips2c_macros.h

```diff
@@ -0,0 +1,17 @@
+#ifndef _MIPS2C_MACROS_H_
+#define _MIPS2C_MACROS_H_
```

Comment:
`MIPS2C_MACROS_H` to avoid the compiler-reserved "underscore, capital letter" name pattern. Not that it matters

Maybe add a comment about what this file is for?

Reply:
Yeah good point, I was copying mm & oot's style but that's not really necessary to follow here.

Added a header comment, and also switched all comments to C89-style for IDO.